[Docs site] Maintenance + PCX-6277 (#8185)
cloudflare/cloudflare-docs: content/fundamentals/api/reference/deprecations.md

Replacement:
Cloudflare Tunnel API

## Account Billing Profile, User Billing Profile, and User Billing History

> **End of life Date: June 6, 2023**

Reviewer: ```suggestion
**End of life date: June 6, 2023**
```

Reviewer: I see that for ##Argo Tunnel above "date" is capitalized but in all the other entries below it's not.

> Cloudflare will no longer continue to support these endpoints due to a system transition and low endpoint usage. There is no replacement for these endpoints. As an alternative, please log in to your Cloudflare account to view your:

Author: ```suggestion
There is no API replacement for these endpoints. As an alternative, please log in to your Cloudflare account to view your:
```
